nipype/interfaces/spm/preprocess.py

```diff
@@ -437,6 +437,251 @@ def _list_outputs(self):
+        desc='List of second order terms to model second derivatives of.')
+    est_unwarp_fwhm = traits.Range(
+        low=0.0, 
+        field='uweoptions.uwfwhm',
+        desc='gaussian smoothing kernel width for unwarp')
+    est_re_est_mov_par = traits.Bool(
+        field='uweoptions.rem',
+        desc='Re-estimate movement parameters at each unwarping iteration.')
+    est_num_of_interations = traits.ListInt(
+        [5], 
+        field='uweoptions.noi',
+        minlen=1, 
+        maxlen=1, 
+        usedfault=True,
+        desc='Number of iterations.')
+    est_taylor_expansion_point = traits.String(
+        'Average', 
+        field='uweoptions.expround',
+        usedefault=True,
+        desc='Point in position space to perform Taylor-expansion around.')
+    reslice_which = traits.ListInt(
+        [2, 1], 
+        field='uwroptions.uwwhich',
+        minlen=2, 
+        maxlen=2, 
+        usedefault=True,
+        desc='determines which images to reslice')
+    reslice_interp = traits.Range(
+        low=0, 
+        high=7, 
+        field='uwroptions.rinterp',
+        desc='degree of b-spline used for interpolation')
+    reslice_wrap = traits.List(
+        traits.Int(), 
+        minlen=3, 
+        maxlen=3,
+        field='uwroptions.wrap',
+        desc='Check if interpolation should wrap in [x,y,z]')
+    reslice_mask = traits.Bool(
+        field='uwroptions.mask',
+        desc='True/False mask output image')
+    out_prefix = traits.String(
+        'u', 
+        field='uwroptions.prefix', 
+        usedefault=True,
+        desc='realigned and unwarped output prefix')
+
+
+class RealignUnwarpOutputSpec(TraitedSpec):
+    mean_image = File(exists=True, desc='Mean image file from the realignment & unwarping')
+    modified_in_files = OutputMultiPath(
+        traits.Either(traits.List(File(exists=True)), File(exists=True)),
+        desc=('Copies of all files passed to '
+              'in_files. Headers will have '
+              'been modified to align all '
+              'images with the first, or '
+              'optionally to first do that, '
+              'extract a mean image, and '
+              're-align to that mean image.'))
+    realigned_unwarped_files = OutputMultiPath(
+        traits.Either(traits.List(File(exists=True)), File(exists=True)),
+        desc='Realigned and unwarped files written to disc.')
+    realignment_parameters = OutputMultiPath(
+        File(exists=True),
+        desc='Estimated translation and rotation parameters')
+
+
+class RealignUnwarp(SPMCommand):
+    """Use spm_uw_estimate for estimating within subject registration and unwarping
+    of time series. Function accepts only one single field map. If in_files is a 
+    list of files they will be treated as separate sessions but associated to the
+    same fieldmap.
+
+    http://www.fil.ion.ucl.ac.uk/spm/doc/manual.pdf#page=31
+
+    Examples
+    --------
+
+    >>> import nipype.interfaces.spm as spm
```

Comment:
you'll have to add `RealignUnwarp` to https://github.com/nipy/nipype/blob/7ea4f1bd3e99a276ea99423a2534a68556e036c7/nipype/interfaces/spm/__init__.py#L8-L10
in order to import this way